reportportal_client/service.py

```diff
@@ -207,7 +207,7 @@ def start_launch(self,
                      mode=None,
                      **kwargs):
         """Start a new launch with the given parameters."""
-        if attributes is not None:
+        if isinstance(attributes, dict):
```

Comment:
I'd rather make a change in the convert function itself.

```
def _dict_to_payload(dictionary):
    """Convert dict to list of dicts.

    :param dictionary: initial dict
    :return list: list of dicts
    """
    if not isinstance(dictionary, dict):
        return None
    system = dictionary.pop("system", False)
    return [
        {"key": key, "value": _convert_string(value), "system": system}
        for key, value in sorted(dictionary.items())
    ]
```

Do not forget about unit tests.

Reply:
if attributes is {}, it does't go the _dict_to_payload function, and directly post to the API with {} value.
do we need re-back the condition to : if attributes is not None  ?